### ISSUE TITLE
Support for Metal: Hellsinger

### ISSUE THREAD

**Issue by jwbruno** [Z1ni/XGP-save-extractor#118]
## Game name: 
Metal: Hellsinger

## Game package name: 
FuncomOsloAS.ProjectHammerhead_pkaskhy6cdq4g

The wgs file seems to contain one container file but it seems that either options available works 94B3912595FE4F2E8F5125A5A875C178:
[wgs.zip](https://github.com/Z1ni/XGP-save-extractor/files/14145335/wgs.zip)
And the Steam save folder contains two files for save %USERPROFILE%\AppData\LocalLow\TheOutsiders\Metal\Save
Including SaveData and SettingsData, and some others, which are very small (close to 10kb)
[Save.zip](https://github.com/Z1ni/XGP-save-extractor/files/14145456/Save.zip)

I tried the `1c1f`, `1cnf` and `1cnf-folder`, that extracts a 70kb file, which seems to be a bit bigger, named: 
 - SaveGame/SaveData
With a binary/json mixed file type. 

It might be the case that it is a Xbox only compatible save file.
Not sure if you faced that before, but I would appreciate any help. 
cheers!

**Comment by Z1ni** [comment]
Yeah, I've seen similar files for Death's Door (#79) and TUNIC (#106), which are also made with Unity, like this one.

There might be a way to split the single save file in the Xbox data to multiple files for the Steam version, but the Steam save data also seems to be encrypted / compressed.
